src/hooks/v2/contractReader/V1ProjectIdOfV2Project.ts

```diff
@@ -1,42 +1,14 @@
 import { BigNumberish, BigNumber } from '@ethersproject/bignumber'
-import { Contract } from '@ethersproject/contracts'
-import { useEffect, useState } from 'react'
-import JBV1TokenPaymentTerminal from '@jbx-protocol/juice-v1-token-terminal/out/JBV1TokenPaymentTerminal.sol/JBV1TokenPaymentTerminal.json'
+import { V2ContractName } from 'models/v2/contracts'
 
-import { JB_V1_TOKEN_PAYMENT_TERMINAL_ADDRESS } from 'constants/contracts'
-
-import { readProvider } from 'constants/readProvider'
-import { readNetwork } from 'constants/networks'
-import { ContractReadResult } from './V2ContractReader'
+import useV2ContractReader, { ContractReadResult } from './V2ContractReader'
 
 export function useV1ProjectIdOfV2Project(
   projectId: BigNumberish | undefined,
 ): ContractReadResult<BigNumber | undefined> {
-  const [loading, setLoading] = useState<boolean>(false)
```

Comment:
Now that we're loading the `JBV1TokenPaymentTerminal` contract in the V2 contract loader, we can use it here :)